src/pykabomu/rina/qpc01.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+from pykabomu.common.abstractions.bytequeue import *
+from abc import ABC, abstractmethod
+
+from pykabomu.common.abstractions.intqueue import *
+from pykabomu.common.exceptions import KabomuException
+from pykabomu.common.helpers import byteutils
+
+# traffic regulator abstract class
+class TrafficRegulator(ABC):
```

Comment:
Change the class name to Qpc101Facility.
```suggestion
class Qpc101Facility(ABC):
```